### PR TITLE
Created Controller for Pensioner Service

### PR DESCRIPTION
**Pull request by Sivaadda**
Created controllers for Pensioner, PensionPlan, Guardian in Pensioner Service.

### CODE REVIEW

**Comment by Anirudh-2024** [comment]
Hi @Sivaadda everything looks good. 
My opinion on the PUT methods. 
PUT method can be simplified by removing the additional get call as the update method is already doing the same thing.

File: PensionManagementPensionerService/Controllers/GuardianController.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            }
+
+        }
+
+        [HttpPut("UpdateGuardianById")]
+        public async Task<IActionResult> UpdateGuardianById(Guid guardianId, [FromBody] GuardianDetails guardianDetails)
+        {
+            try
+            {
+                var result = await _guardianRepository.GetGuardianById(guardianId);
```
Comment (Anirudh-2024) [inline PensionManagementPensionerService/Controllers/GuardianController.cs]: request you to look into this please i felt like this is an additional call

Comment (Anirudh-2024) [reply PensionManagementPensionerService/Controllers/GuardianController.cs]: same goes to the other 2 controllers as well

File: PensionManagementPensionerService/Controllers/GuardianController.cs [hunk 2]
```diff
@@ -65,9 +65,8 @@ public async Task<IActionResult> UpdateGuardianById(Guid guardianId, [FromBody]
         {
             try
             {
-                var result = await _guardianRepository.GetGuardianById(guardianId);
                 await _guardianRepository.UpdateGuardianById(guardianId, guardianDetails);
-                return Ok(result);
+                return Ok(guardianDetails);
```
Comment (Anirudh-2024) [inline PensionManagementPensionerService/Controllers/GuardianController.cs]: is your update method not returning the output?

**Review by Anirudh-2024 — CHANGES_REQUESTED** [review]

File: PensionManagementPensionerService/Controllers/GuardianController.cs (hunk 2 above)
Comment (Sivaadda) [reply PensionManagementPensionerService/Controllers/GuardianController.cs]: It is returning the guardian Details which we updated.
The result variable is checking guardian id as we removed that line of code. so in place of result I replaced it with guardian details.
Both will display same output.

**Review by Anirudh-2024 — APPROVED** [review]
LGTM

**Pull request merged**